### DISCUSSION TITLE
help develop

### DISCUSSION THREAD

**Discussion by lublak** [drakkan/sftpgo#921] [General]
I think the project is really good.
The question is whether it can help with other things besides the money (which is still gladly done).
Unfortunately, I'm not that proficient in golang or sftp/webdav/ftp etc.
The issues that are currently open are mostly incomprehensible to me and could not help with coding here.
Therefore: Is there anything I can do to support you additionally?

**Comment by drakkan** [comment]
Hi,

thanks for your support, much appreciated. Please read this [post](https://words.filippo.io/pay-maintainers/). I agree with it. In particular this part:

> However, new contributors increase the maintainer's workload without improving their motivation or sustainability. I often read advice along the lines of "if you want to help a project send PRs" and I never understood it. Reviewing and iterating or pushing back on PRs is work! Often more than writing the code itself.

Other useful things are testing resources. 

For example take a look at #893. Testing SFTPGo with Amazon RDS for MySQL, aside from the time, takes some money. If someone could provide these non-free resources to the project they would be useful.
Although in general users shouldn't behave like the person who reported that particular bug. They should provide the project with the resources required to reproduce their issue instead of just opening an issue and then disappearing.
